admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -122,6 +124,8 @@ These are:
 .. note::
    Please note, the commands have to be run in this order.
 
+.. warning:: If you already have Basic encryption enabled - you **must not** enable Master Key Based encryption.
```

Comment:
um, enabling the master key based encryption requires these steps as well. see the next section below. I agree that we need to make clearer that you cant switch between the two without decrypting the files first.

Did you run into problems trying this? Is there a github issue in core?